Main.java

```diff
@@ -9,7 +10,22 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+// create a Scanner for user input
+ Scanner input = new Scanner(System.in);
+
+ // declare a variable to store Celsius
+
+ // ask the user for celsius
+  System.out.println("Please enter the Celsius that you want to change to Fahrenheit:");
+  // initialize the celsius that needs to be converted to Fahrenheit
+
+
+// declare and initialize a variable for Celsius
+int Celsius = input.nextInt();
+
+// delcare and initialize a variable for Fahrenheit
+int Fahrenheit = ((Celsius*9)/5 +32);
+ // tell the user the converted temperature
+ System.out.println("The answer is "  + Fahrenheit + "F.");

```

Comment:
Good work with the math and comments. For your variables, remember that they should always begin with a lowercase letter. Also, it didn't pass the test cases because your output did not match exactly what the sample showed. In particular, you have a colon in the output on line 19 that wasn't there, and your final output on line 29 is also different.